src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -10,8 +10,12 @@ pub type nlink_t = u64;
 pub type blksize_t = i64;
 pub type suseconds_t = i64;
 pub type __u64 = ::c_ulong;
+pub type gregset_t = [::c_ulong; 48];
+pub type fpregset_t = [::c_double; 33];
 
 s! {
+    pub struct pt_regs {};
```

Comment:
**Q**: What's the best way to workaround the issue of declaring this struct with the `s!` macro? Because the macro does not take in field-less struct, it would yield following error:

```
error: no rules expected the token `;`
  --> src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs:17:23
   |
17 |     pub struct pt_regs {};
   |                       ^ no rules expected this token in macro call
   | 
  ::: src/macros.rs:66:1
   |
66 | macro_rules! s {
   | -------------- when calling this macro
```


 I am thinking of manually copy some of the guts of the `s!` to declare this struct though I do not think it is the way. Any guidance is greatly appreciated cc @gnzlbg @Susurrus